packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -432,7 +498,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   // stylelint-disable plugin/selector-bem-pattern
   .mdc-textfield__input,
   .mdc-textfield__label,
-  + .mdc-textfield-helptext {
+  .mdc-textfield-helptext {
```

Comment:
Why was the `+` here removed? Isn't helptext still a sibling to the entire textfield?